### PR TITLE
#1055 added test for termination of orders/wbs

### PR DESCRIPTION
**Pull request by krzyk**
#1055 
- added test for termination of orders/WBS in case github link is removed
- added todo for implementation

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#1220](https://github.com/zerocracy/farm/issues/1220) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#1220](https://github.com/zerocracy/farm/issues/1220) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/1731385975); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/main/java/com/zerocracy/pm/Claims.java [hunk 1]
```diff
@@ -44,10 +44,10 @@
  * @author Yegor Bugayenko (yegor256@gmail.com)
  * @version $Id$
  * @since 0.9
- * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
  * @todo #1208:30min During #1208 fixing PMO claims was saved locally
  *  and removed from production. We need to analyze it, find critical claims
  *  and submit them to current `claims.xml` in PMO.
+ * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
```
Comment (carlosmiranda) [inline src/main/java/com/zerocracy/pm/Claims.java]: @krzyk looks like an unrelated change. Let's remove it.

File: src/test/resources/com/zerocracy/bundles.terminates_orders_wbs_on_link_removal/_after.groovy [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.zerocracy.bundles.terminates_orders_wbs_on_link_removal
+
+import com.jcabi.xml.XML
+import com.zerocracy.Project
+
+// @todo #1055:30min Implement logic for removal of all orders and clear WBS
```
Comment (carlosmiranda) [inline src/test/resources/com/zerocracy/bundles.terminates_orders_wbs_on_link_removal/_after.groovy]: @krzyk let's move the puzzle to `terminate_orders_wbs_on_link_removal`. That class is empty and it's unclear why, while we have some commented code here so it's more obvious that there's something missing.

File: src/main/java/com/zerocracy/pm/Claims.java (hunk 1 above)
Comment (krzyk) [reply src/main/java/com/zerocracy/pm/Claims.java]: @carlosmiranda I can't because the master breaks. So depending which PR will be merged first it will fail with rultor when merging master :( I have this change in each PR

Comment (carlosmiranda) [reply src/main/java/com/zerocracy/pm/Claims.java]: @krzyk right, makes sense

**Comment by krzyk** [comment]
@carlosmiranda updated, please check

**Comment by g4s8** [comment]
@rultor merge

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#1220](https://github.com/zerocracy/farm/issues/1220) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by ypshenychka** [comment]
@carlosmiranda According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only one issue was found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by carlosmiranda** [comment]
@ypshenychka confirmed

**Comment by ypshenychka** [comment]
@carlosmiranda Thank you

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +15 point(s) just awarded to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)
